dev/start

```diff
@@ -83,11 +87,18 @@ if [[ $ENABLE_AUTHN_OIDC = true ]]; then
   echo "Configuring Keycloak as OpenID provider for automatic testing"
   docker-compose exec conjur conjurctl policy load cucumber /src/conjur-server/ci/authn-oidc/policy.yml
 
-  enabled_authenticators="$enabled_authenticators,authn-oidc/okta,authn-oidc/keycloak"
+  enabled_authenticators="$enabled_authenticators,authn-oidc/okta,authn-oidc/keycloak,authn-oidc/adfs"
```

Comment:
The addition of `authn-oidc/adfs` can actually move inside the if clause below

Reply:
Done